scripts/build

```diff
@@ -5,5 +5,5 @@ source $(dirname $0)/version
 
 cd $(dirname $0)/..
 mkdir -p bin
-echo Building submariner-engine version $VERSION
-CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION" -o bin/submariner-engine main.go
\ No newline at end of file
+echo Building coastguard version $VERSION
+CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION" -o bin/coastguard pkg/coastguard/main.go
```

Comment:
coastguard-controller would be more appropriate for the binary??

Reply:
Yes, feel free to change in your later patch. Thanks Janki